### ISSUE TITLE
Input via stdin doesn't work for large templates

### ISSUE THREAD

**Issue by Dunedan** [duncanhall/expand-swagger-refs#2]
Thanks for implementing stdin and stdout in such a timely manner.

There still seems to be a bug: If you pass a swagger template with more than 65535 characters to `swagger-expand` via stdin it fails with an error message.
The exact error message varies based on the data provided (as depending on its length it gets corrupted differently), but it's always related to the data not being compliant with JSON anymore.
E.g. with https://api.zalando.com/schema/swagger.json, I get:
```
foo@bar:~$ swagger-expand /tmp/swagger.json
Expanded file written to /tmp/swagger.json-expanded
foo@bar:~$$ swagger-expand < /tmp/swagger.json 
undefined:1322
                        "description",: "Category Key response.",
                                     ^

SyntaxError: Unexpected token , in JSON at position 65520
    at JSON.parse (<anonymous>)
    at ReadStream.<anonymous> (~/node_modules/expand-swagger-refs/bin/from-stdin.js:14:29)
    at emitNone (events.js:91:20)
    at ReadStream.emit (events.js:188:7)
    at endReadableNT (_stream_readable.js:975:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

NodeJS version used is 7.7.4.

**Comment by duncanhall** [comment]
Thanks, I'll look into this soon.

**Comment by duncanhall** [comment]
This is now fixed in version `2.0.1`. Can you `npm update` and let me know if you are still having issues?

**Comment by Dunedan** [comment]
Works like a charm. Thanks a lot.
